### DISCUSSION TITLE
How does saving work with HTML5 export?

### DISCUSSION THREAD

**Discussion by ghost** [HaxeFlixel/flixel#2351] [Q&A]
I read in the docs there is an API for saving to local shared object very similar to the flash API, but is that API only for flash swf export? If I did an HTML5 export, can I still make a local shared object rather than have to make localstorage or indexeddb?

**Comment by Geokureli** [comment]
`FlxG.save` should work on all targets
https://haxeflixel.com/documentation/flxsave/
